tools/mo/openvino/tools/mo/middle/layer_normalization.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from openvino.tools.mo.front.common.partial_infer.utils import int64_array
+from openvino.tools.mo.front.tf.graph_utils import create_op_node_with_second_input
+from openvino.tools.mo.graph.graph import Graph, rename_nodes
+from openvino.tools.mo.middle.replacement import MiddleReplacementPattern
+from openvino.tools.mo.ops.elementwise import Mul, Add
+from openvino.tools.mo.ops.mvn import MVN
+
+
+class LayerNormalization(MiddleReplacementPattern):
+    """
+    Decompose LayerNorm(x) to MVN(x) * gamma + beta
+    """
+    enabled = True
+
+    def find_and_replace_pattern(self, graph: Graph):
+        for node in graph.get_op_nodes(op='LayerNorm'):
+            node_name = node.soft_get('name', node.id)
+            assert node.has_valid('axis'), 'Incorrect axis value {} for the node {}'.format(node.axis, node_name)
```

Comment:
This assertion is faulty. If the node has no 'axis' the exception will generate another exception when will try to access node.axis